Gemfile.lock

```diff
@@ -9,6 +9,13 @@ PATH
   specs:
     card-mod-csv_import (0.8)
       card
+      card-mod-layout
```

Comment:
Semaphore is saying you need to update Gemfile.lock. Looks like you already did, but maybe try again?